syft/workers/base.py

```diff
@@ -462,7 +471,11 @@ def execute_command(self, message: tuple) -> PointerTensor:
                 response = hook_args.register_response(
                     command_name, response, list(return_ids), self
                 )
-                return response
+                if return_value or isinstance(response, (int, float, bool, str)):
```

Comment:
Does this mean I can set `return_value` to `False` and still get a response? That seems surprising.